app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2017 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.global.db
+
+import android.arch.persistence.room.Database
+import android.arch.persistence.room.RoomDatabase
+import com.duckduckgo.app.httpsupgrade.db.HTTPSUpgradeDomain
+import com.duckduckgo.app.httpsupgrade.db.HTTPSUpgradeDomainDAO
+
+@Database(entities = arrayOf(HTTPSUpgradeDomain::class), exportSchema = false, version = 1)
```

Comment:
we'd probably be as well exporting the schema? it would let us write migration tests when we next need to upgrade the DB.